src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx

```diff
@@ -665,8 +664,7 @@ For users on the [New Relic One user model](/docs/accounts/original-accounts-bil
       </td>
 
       <td>
-        Basic users, no matter what group they're assigned to, **always** have [basic user](#user-type) abilities, no more and no less. 
-
+        When basic users are added to a group, that group's role-related restrictions apply, but a basic user can never be granted more capabilities than they start with.
```

Comment:
Suggestion: "When basic users are added to a group, that group's role-related restrictions apply. What's more, a basic user can never be granted more capabilities than they start with."